kyo-core/shared/src/main/scala/kyo/internal/FileImplicits.scala

```diff
@@ -0,0 +1,21 @@
+package kyo.internal
+
+import scala.language.implicitConversions
+import scala.quoted.*
+
+opaque type FileNameWithLine <: String = String
```

Comment:
What do you think about `Position` instead of `FileNameWithLine`?

Reply:
My only slight fear is that we are colliding with Scala's `quotes.reflect.Position`... perhaps `FilePosition`?

I do think `internal.Position` is much cleaner and concise in function definitions. We can proceed with it for now.